src/utils/delta_btree_map/src/lib.rs

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#![feature(btree_cursors)]
```

Comment:
The same as https://github.com/risingwavelabs/risingwave/pull/14204/files#r1436788805.

IIUC, methods related to the feature is not directly used in this crate, and therefore the feature is not required here. Only the `delta_btree_map` crate that uses the related feature needs to explicitly enable the feature.

Reply:
Actually it is directly used😇

https://github.com/risingwavelabs/risingwave/blob/d27d6c1eca115c60a6b31958e4a8ef6de5a5affe/src/common/src/estimate_size/collections/btreemap.rs#L95